CHANGELOG.md

```diff
@@ -40,6 +40,9 @@
 
 ## Other changes
 
+- The deprecated legacy interface to extract to EasyCrypt the been removed
```

Comment:
```suggestion
- The deprecated legacy interface to extract to EasyCrypt has been removed
```

Reply:
Oups. Sorry about that. Thanks for the careful review.

Reply:
My added value is over 9000!